beagle/settings.py

```diff
@@ -278,6 +278,7 @@
 
 POOLED_NORMAL_FILE_GROUP = os.environ.get('BEAGLE_POOLED_NORMAL_FILE_GROUP', '1552617b-3b06-46de-921a-a000a9129385')
 
+CELERY_TASK_ALWAYS_EAGER=DEBUG
```

Comment:
@aef- How does this work? What behavior does it introduce? sounds useful

Reply:
I updated it to use Celery 4 settings but it's exactly what you did in the test files.